app/entrypoint.sh

```diff
@@ -45,6 +45,19 @@ function get_nginx_proxy_cid {
     fi
 }
 
+function get_nginx_gen_cid {
+    # Check if any container has been labelled as the nginx gen container.
+    local labeled_cid=$(docker_api "/containers/json" | jq -r '.[] | select( .Labels["com.github.jrcs.letsencrypt_nginx_proxy_companion.nginx_gen"] == "true")|.Id')
+    if [[ ! -z "${labeled_cid:-}" ]]; then
+        export NGINX_DOCKER_GEN_CONTAINER=$labeled_cid
+    fi
+    if [[ -z "${NGINX_DOCKER_GEN_CONTAINER:-}" ]]; then
```

Comment:
Remove this condition to make it optional, don't error out. Document it instead.